introductory-tutorials/intro-to-julia/04. Loops.ipynb

```diff
@@ -103,6 +103,8 @@
    "source": [
     "Now let's use `for` loops to create some addition tables, where the value of every entry is the sum of its row and column indices. <br>\n",
     "\n",
+    "For more information about fast indexing of multidimensional arrays inside nested loops can be found at https://docs.julialang.org/en/v1/manual/performance-tips/#Access-arrays-in-memory-order,-along-columns-1",
```

Comment:
```suggestion
    "Note that we iterate over this array via column-major loops in order to get the best performance. More information about fast indexing of multidimensional arrays inside nested loops can be found at https://docs.julialang.org/en/v1/manual/performance-tips/#Access-arrays-in-memory-order,-along-columns-1",
```

Reply:
LGTM as well, and I like your suggestion Nathan!

Reply:
Applied manually in e61c7f9.